modules/nf-core/mageck/test/main.nf

```diff
@@ -1,8 +1,8 @@
 process MAGECK_TEST {
-    tag "$meta.id"
+    tag "${meta.treatment}_${meta.reference}"
```

Comment:
Ohhh, we don't like adding new meta map elements into modules! Is this necessary?